scripts/generate-renode-scripts.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+
+import csv
+import zlib
+import argparse
+
+def generate_ethmac(data):
+    base = get_entry_value(data, 'csr_base', 'ethmac')
+    buffer_base = get_entry_value(data, 'memory_region', 'ethmac')
+    buffer_size = get_entry_value(data, 'memory_region', 'ethmac', 1)
+    interrupt = get_entry_value(data, 'constant', 'ethmac_interrupt')
+
+    result = [
```

Comment:
Use a triple quoted string for long output like this.

Reply:
This is reworked to return a single, multiline string.